--- phase3/arti_kmp_ffi.android.kt
+++ final/arti_kmp_ffi.android.kt
@@ -599,6 +599,8 @@
 }
 
 // Public interface members begin here.
+
+
 
 
 
@@ -1280,6 +1282,9 @@
             FfiConverterTypeBridgesEnabled.read(buf),
             FfiConverterOptionalString.read(buf),
             FfiConverterOptionalString.read(buf),
+            FfiConverterBoolean.read(buf),
+            FfiConverterLong.read(buf),
+            FfiConverterLong.read(buf),
         )
     }
 
@@ -1289,7 +1294,10 @@
             FfiConverterSequenceString.allocationSize(value.`bridges`) +
             FfiConverterTypeBridgesEnabled.allocationSize(value.`bridgesEnabled`) +
             FfiConverterOptionalString.allocationSize(value.`stateDir`) +
-            FfiConverterOptionalString.allocationSize(value.`cacheDir`)
+            FfiConverterOptionalString.allocationSize(value.`cacheDir`) +
+            FfiConverterBoolean.allocationSize(value.`allowOnionAddrs`) +
+            FfiConverterLong.allocationSize(value.`connectTimeoutNanos`) +
+            FfiConverterLong.allocationSize(value.`resolveTimeoutNanos`)
     )
 
     override fun write(value: ArtiConfig, buf: ByteBuffer) {
@@ -1299,6 +1307,9 @@
             FfiConverterTypeBridgesEnabled.write(value.`bridgesEnabled`, buf)
             FfiConverterOptionalString.write(value.`stateDir`, buf)
             FfiConverterOptionalString.write(value.`cacheDir`, buf)
+            FfiConverterBoolean.write(value.`allowOnionAddrs`, buf)
+            FfiConverterLong.write(value.`connectTimeoutNanos`, buf)
+            FfiConverterLong.write(value.`resolveTimeoutNanos`, buf)
     }
 }
 
